src/features/surveys/pages/PublicSurveyPage.tsx

```diff
@@ -18,23 +18,23 @@ import {
 // @ts-ignore
 import { useFormState } from 'react-dom';
 
+import { Msg, useMessages } from 'core/i18n';
 import SurveyForm from 'features/surveys/components/SurveyForm';
-import { submit } from '../actions/submit';
+import useIsMobile from 'utils/hooks/useIsMobile';
 import {
   ZetkinSurveyExtended,
   ZetkinSurveyFormStatus,
   ZetkinSurveySignatureType,
   ZetkinUser,
 } from 'utils/types/zetkin';
+import ZUIAlert from 'zui/components/ZUIAlert';
+import ZUIButton from 'zui/components/ZUIButton';
+import ZUILink from 'zui/components/ZUILink';
+import ZUIPublicSurveyOption from 'zui/components/ZUIPublicSurveyOption';
```

Comment:
All of the changes to imports seem unnecessary to me. If you're not sure how they happened, please have a look in your editor in case it does some sort of automatic sorting of imports, or something like that, and disable it for future PRs. 😊